mLRS/Common/bind.h

```diff
@@ -83,7 +84,8 @@ class tBindBase
     int8_t pressed_cnt;
 
   private:
-    tSxGlobalConfig* gconfig;  
+    tSxGlobalConfig* gconfig;
+    bool last_was_7x = false;
```

Comment:
hm ... this I do not like, as it brings in stateness ,,,
let's see what we can do about it